requirements.txt

```
@@ -8,7 +8,7 @@ pathlib2==2.3.2; python_version<"3"
 prov==1.5.1
 bagit==1.6.4
 mypy-extensions
-psutil
+psutil<=5.6.1
```

If it was a bad release then we should ban just that release, and not all newer releases as well: 

```suggestion
psutil != 5.6.1
```